browser/ipfs/test/ipfs_service_browsertest.cc

```diff
@@ -1666,4 +1676,24 @@ IN_PROC_BROWSER_TEST_F(IpfsServiceBrowserTest, DNSResolversConfig) {
   }
 }
 
+IN_PROC_BROWSER_TEST_F(IpfsServiceBrowserTest, MigrateProfilePrefs) {
+  auto* prefs = browser()->profile()->GetPrefs();
```

Comment:
Might be worth checking the `false` `false` and `true` `true` cases too, just to make sure we've covered everything.